<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalDat</class>
 <widget class="QDialog" name="CalDat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Калькулятор дат</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QDateEdit" name="firstDate"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Количество дней:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Проверка года на високосность:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="difButton">
        <property name="text">
         <string>Вычислить</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Вычисление даты по количеству дней:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="sumDays">
        <property name="text">
         <string>Вычислить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QLabel" name="difDays">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPushButton" name="checkYear">
        <property name="text">
         <string>Проверить</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLineEdit" name="year"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Исходная дата:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Получившаяся дата:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDateEdit" name="defaDate"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="numDays"/>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Разница между датами в днях:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDateEdit" name="secondDate"/>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="sumDayslab">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
